src/sentry/static/sentry/app/components/button.jsx

```diff
@@ -277,12 +288,13 @@ const getLabelPadding = ({size, priority, borderless}) => {
   }
 };
 
-const ButtonLabel = styled(({size, priority, borderless, ...props}) => (
+const ButtonLabel = styled(({size, priority, borderless, align, ...props}) => (
   <span {...props} />
 ))`
-  display: flex;
+  display: grid;
```

Comment:
Is this gonna change the layout of existing buttons? /cc @Chrissy

Reply:
I had a hard time getting alignment working otherwise :/

Reply:
yeah i think this is gonna break some stuff since flex and grid flow differently. 

It won't break any layouts since the two have a `block` level formation—it would mostly effect any child layouts we pass to this label. Definitely wanna fire up Percy again and check in on this.